### DISCUSSION TITLE
Increase logging level of Elasticsearch components.

### DISCUSSION THREAD

**Discussion by john-mathew-itt** [Security-Onion-Solutions/securityonion#6507] [Unsupported Versions] [answered]
Hi Community,

How can I increase the logging level of Elasticsearch components like Logstash, Elasticsearch and Kibana in Security Onion 2?  What is the right way to increase the logging level so that I can debug issues related to the ELK components? What I have already tried is editing `/opt/so/conf/elasticsearch/log4j2.properties` file changing `rootLogger.level = debug` and executing `so-elasticsearch-restart` but didn't help. The logs still have debugging level of **info**.

**Security Onion version:** 2.3.90
Is this a cloud deployment or on-prem?
**On-prem deployment**
Did you install from our Security Onion ISO image or did you perform a network installation?
**Official ISO image**

**Comment by weslambert** [comment, marked as answer]
HI @john-mathew-itt ,

You might want to try copying `/opt/so/saltstack/default/salt/elasticsearch/files/log4j2.properties` to `/opt/so/saltstack/local/salt/elasticsearch/files/log4j2.properties`, make your changes there, then restart.

Thanks,
Wes

**Comment by john-mathew-itt** [comment]
Thanks @weslambert This worked perfectly.
